src/particles/beam/BeamParticleContainer.cpp

```diff
@@ -414,6 +416,7 @@ BeamParticleContainer::InSituComputeDiags (int islice, int islice_local)
     m_insitu_rdata[islice+14*m_nslices] = amrex::get<14>(a)*sum_w_inv;
     m_insitu_rdata[islice+15*m_nslices] = amrex::get<15>(a)*sum_w_inv;
     m_insitu_rdata[islice+16*m_nslices] = amrex::get<16>(a)*sum_w_inv;
+    m_insitu_rdata[islice+17*m_nslices] = amrex::get<17>(a)*sum_w_inv;
     m_insitu_idata[islice             ] = amrex::get<17>(a);
```

Comment:
Shouldn't this be next one?
```suggestion
    m_insitu_idata[islice             ] = amrex::get<18>(a);
```

Reply:
Yes